documentation/v2/body.html

```diff
@@ -106,7 +116,7 @@
       </section>
       <section id="source-maps">
         <%= htmlFor('source_maps') %>
-      </section>
+      </section>      
```

Comment:
Do you mind please giving me write access to this branch? There are lots of little things like this trailing whitespace, and misspellings, that are easier to just fix than point out in comments.